src/ui/utils/devtools-toolbox.ts

```diff
@@ -57,6 +58,18 @@ export class DevToolsToolbox {
   async init(selectedPanel: StartablePanelName) {
     await this.threadFront.initializeToolbox();
 
+    const testName = getTest();
+    if (testName) {
+      await gToolbox.selectTool("debugger");
+
+      window.Test = await import("test/harness");
+
+      const script = document.createElement("script");
+      script.src = `/test/scripts/${testName}`;
+
+      document.head.appendChild(script);
+    }
```

Comment:
I'm not sure this is the best place for it to live _either_ but since this is the place that calls `threadFront.initializeToolbox()` I moved it here for now. At least it runs at the same time as before.

Baby steps!